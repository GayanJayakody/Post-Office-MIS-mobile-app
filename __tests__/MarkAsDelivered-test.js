import React from 'react';
import Delivered from '../screens/MarkAsDelivered';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<Delivered />).toJSON();
  expect(tree).toMatchSnapshot();
});