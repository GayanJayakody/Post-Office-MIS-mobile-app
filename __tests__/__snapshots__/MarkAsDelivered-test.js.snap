// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    allowCaptureRetake={false}
    colorForScannerFrame="black"
    frameColor="yellow"
    laserColor="blue"
    onReadCode={[Function]}
    scanBarcode={true}
    showFrame={false}
    style={
      Object {
        "backgroundColor": "black",
        "flex": 1,
      }
    }
  >
    <RCTSafeAreaView
      emulateUnlessSupported={true}
      style={
        Object {
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "space-between",
          "paddingBottom": 0,
          "paddingTop": 8,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
            "paddingHorizontal": 15,
          }
        }
      >
        <Image
          resizeMode="contain"
          style={
            Object {
              "flex": 1,
              "justifyContent": "center",
            }
          }
        />
      </View>
    </RCTSafeAreaView>
    <View
      style={
        Object {
          "flex": 10,
          "flexDirection": "column",
        }
      }
    >
      <CKCamera
        cameraOptions={
          Object {
            "flashMode": "auto",
            "focusMode": "on",
            "ratioOverlayColor": undefined,
            "zoomMode": "on",
          }
        }
        frameColor="yellow"
        laserColor="blue"
        onReadCode={[Function]}
        scanBarcode={true}
        scannerOptions={
          Object {
            "colorForFrame": 4278190080,
            "frameHeight": 200,
            "offsetFrame": 30,
          }
        }
        showFrame={false}
        style={
          Object {
            "flex": 1,
            "justifyContent": "flex-end",
          }
        }
      />
    </View>
    <RCTSafeAreaView
      emulateUnlessSupported={true}
      style={
        Array [
          Object {
            "flex": 2,
            "flexDirection": "row",
            "justifyContent": "space-between",
            "padding": 14,
          },
          Object {
            "backgroundColor": "#ffffff00",
          },
        ]
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "flex-start",
            "opacity": 1,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
              "fontSize": 20,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "flex-end",
            "padding": 10,
          }
        }
      />
    </RCTSafeAreaView>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "lightblue",
        "borderBottomLeftRadius": 20,
        "borderBottomRightRadius": 20,
        "borderTopLeftRadius": 20,
        "borderTopRightRadius": 20,
        "height": 200,
        "justifyContent": "center",
        "textAlign": "center",
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "royalblue",
          "fontSize": 25,
        }
      }
    >
      Letter/Parcel ID
    </Text>
    <Text
      style={
        Object {
          "color": "royalblue",
          "fontSize": 25,
        }
      }
    />
    <Text
      style={
        Object {
          "fontSize": 5,
        }
      }
    >
      

    </Text>
    <View
      accessibilityRole="button"
      accessibilityState={Object {}}
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "margin": 8,
                "textAlign": "center",
              },
            ]
          }
        >
                      Delivered            
        </Text>
      </View>
    </View>
    <Text
      style={
        Object {
          "fontSize": 5,
        }
      }
    >
      


    </Text>
    <View
      accessibilityRole="button"
      accessibilityState={Object {}}
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "margin": 8,
                "textAlign": "center",
              },
            ]
          }
        >
                  Receiver Unavailable        
        </Text>
      </View>
    </View>
    <Text
      style={
        Object {
          "fontSize": 5,
        }
      }
    >
      

    </Text>
  </View>
</View>
`;
